src/model/Global.ts

```diff
@@ -417,6 +422,25 @@ class GlobalModel extends Model<ECUnitOption> {
         return this._theme;
     }
 
+    getLocale(): LocaleOption {
+        return this._locale;
+    }
+
+    getWithLocale(localePosition: Array<string>, optionsPosition?: Array<string>, localeHandlerFn?: (text: string) => string): any {
+        // console.log(optionsPosition, localePosition);
+        const locale = this.getLocale()
+        let localeText: string | any;
+        localePosition.map(t => {
+            localeText = localeText ? localeText[t] : locale[t];
+        })
+
+        if(localeHandlerFn) {
+            localeText =  localeHandlerFn(localeText);
```

Comment:
Extra space after '=' here.

Also, I'm not sure the last parameter `localeHandleFn` is necessary. It will increase the complexity of using this function.

Reply:
sometimes the text may  handle after get，for example:
```
lang: ecModel.getWithLocale(['toolbox', 'saveAsImage', 'lang'], null, (t) => t.slice())
```
the ```toolebox.saveAsImage.lang``` is an array but need to be ```slice()``` after gotten.